src/Gallery/Api/index.ts

```diff
@@ -15,6 +15,7 @@ export type GalleryApi = {
   upload: (fileData: FormData) => Promise<GalleryItem>;
   save: (details: GalleryItem) => Promise<{ item: GalleryItem }>;
   load: (id: string) => Promise<{ item: GalleryItem }>;
+  //TODO : create api method to get svg string/object loadSvg: (id: string) => Promise<{ item: GalleryItem }>;
```

Comment:
Here I am not sure if we are going to send from API the actual SVG or a gallery object. If we load same as pictures, then  I think we should send the SVG directly.